### PR TITLE
feat: tweak table dropping mechanism

### PR DESCRIPTION
**Pull request by IlyaFaer**
The PR requires changes of the [#277](https://github.com/googleapis/python-spanner/pull/277) to be merged first.

### CODE REVIEW

**Comment by IlyaFaer** [comment]
@larkee, I've been using this method from the beginning. In fact most of the testing cleanups will fail without these changes - meaning all the screenshots with test logs, I've been sending you during the last month, were based on these changes.

**Review by larkee — COMMENTED** [review]
LGTM although I would prefer a couple unit tests for this.

Also, the `compliance_test` is failing to run:
```
INTERNALERROR> sqlalchemy.exc.ArgumentError: Could not parse rfc1738 URL from string 'None'
```
Any ideas on what is causing this break?

**Review by larkee — COMMENTED** [review]
The break was caused by #25 but has now been resolved.

File: google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py [hunk 1]
```diff
@@ -86,6 +86,35 @@ def visit_empty_set_expr(self, type_):
...
+
+        Args:
+            (sqlalchemy.schema.DropTable): DROP TABLE statement object.
+
+        Returns:
+            str:
+                DDLs separated by semicolons, which will sequentially
+                drop indexes, foreign keys constraints and then the
+                table itself.
+        """
```
[the remark this answers is not in the document]
Comment (IlyaFaer) [reply google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py:105]: Okay, pushed

**Review by larkee — APPROVED** [review]

**Pull request merged**